alphastats/gui/utils/analysis_helper.py

```diff
@@ -37,7 +37,7 @@ def save_plot_to_session_state(plot, method):
 
 
 def display_df(df):
```

Comment:
This should use pandas stylers, rather than changing the data, as this can become quite a lot to handle. Also not sure, why we need this.